organization.md

```diff
@@ -10,10 +10,12 @@ This section shows how developers can make use of the features of this specifica
 ```json
 {
     "id": "01fc7fce-bf78-4b50-903a-fb3b760f5b30",
+    "netsuite_id": "999999",
```

Comment:
lets call this "internal_id" that way if we move off NetSuite its not tied to that system.